test/e2e/scenario/telemetry.scenario.ts

```diff
@@ -37,11 +36,41 @@ describe('telemetry', () => {
         window.DD_RUM!.addAction('hop', context as any)
       })
       await flushEvents()
-      expect(serverEvents.telemetry.length).toBe(1)
-      const event = serverEvents.telemetry[0] as TelemetryErrorEvent
+      expect(serverEvents.telemetryErrors.length).toBe(1)
+      const event = serverEvents.telemetryErrors[0]
       expect(event.telemetry.message).toBe('bar')
       expect(event.telemetry.error!.kind).toBe('Error')
       expect(event.telemetry.status).toBe('error')
       serverEvents.empty()
     })
+
+  createTest('send init configuration for logs')
+    .withSetup(bundleSetup)
+    .withLogs({
+      enableExperimentalFeatures: ['telemetry_configuration'],
+      forwardErrorsToLogs: true,
+    })
+    .run(async ({ serverEvents }) => {
+      await flushEvents()
+      expect(serverEvents.telemetryConfigurations.length).toBe(1)
+      const event = serverEvents.telemetryConfigurations[0]
+      expect(event.telemetry.type).toBe('configuration')
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏testing that the type is `configuration` is a bit redundant with the fact that we have already filtered the telemetry events on this value